### ISSUE TITLE
 peer-to-peer code reviews:Awesome-Book-ES6

### ISSUE THREAD

**Issue by BouaikIlham** [abiodunraheem/Awesome-Book-ES6#2]
Good job so far but we found some issues :
The page should be deployed for easy accessibility .
The setup instruction in the readme file should be very clear and provide links and steps for cloning incase the user wants to clone the repository .
